var/spack/repos/builtin/packages/pmix/package.py

```diff
@@ -9,43 +9,43 @@
 
 
 class Pmix(AutotoolsPackage):
-    """The Process Management Interface (PMI) has been used for quite some time
-       as a means of exchanging wireup information needed for interprocess
-       communication. Two versions (PMI-1 and PMI-2) have been released as part
-       of the MPICH effort. While PMI-2 demonstrates better scaling properties
-       than its PMI-1 predecessor, attaining rapid launch and wireup of the
-       roughly 1M processes executing across 100k nodes expected for exascale
-       operations remains challenging.  PMI Exascale (PMIx) represents an
-        attempt to resolve these questions by providing an extended version
-       of the PMI definitions specifically designed to support clusters up
-       to and including exascale sizes.  The overall objective of the project
-       is not to branch the existing definitions - in fact, PMIx fully
-       supports both of the existing PMI-1 and PMI-2 APIs - but rather to
-       (a) augment and extend those APIs to eliminate some current restrictions
-       that impact scalability, (b) establish a standards-like body for
-       maintaining the definitions, and (c) provide a reference implementation
-       of the PMIx standard that demonstrates the desired level of
-       scalability."""
+    """The Process Management Interface (PMI) has been used for quite some
+       time as a means of exchanging wireup information needed for
+       interprocess communication. However, meeting the significant
+       orchestration challenges presented by exascale systems requires
+       that the process-to-system interface evolve to permit a tighter
+       integration between the different components of the parallel
+       application and existing and future SMS solutions.
 
-    homepage = "https://pmix.github.io/pmix"
-    url      = "https://github.com/pmix/pmix/releases/download/v2.0.1/pmix-2.0.1.tar.bz2"
+       PMI Exascale (PMIx) addresses these needs by providing an extended
+       version of the PMI definitions specifically designed to support
+       exascale and beyond environments by: (a) adding flexibility to the
+       functionality expressed in the existing APIs, (b) augmenting the
+       interfaces with new APIs that provide extended capabilities, (c)
+       forging a collaboration between subsystem providers including
+       resource manager, fabric, file system, and programming library
+       developers, (d) establishing a standards-like body for maintaining
+       the definitions, and (e) providing a reference implementation of the
+       PMIx standard that demonstrates the desired level of scalability
+       while maintaining strict separation between it and the standard
+       itself."""
 
+    homepage = "https://pmix.org"
+    url      = "https://github.com/pmix/pmix/releases/download/v3.1.2/pmix-3.1.2.tar.bz2"
+
```

Comment:
one last thing: do you want to be listed here as a `maintainers = ['rhc54']` entry? The idea is to ping the people mentioned here on (tricky) updates.

Reply:
sure - I have no issue maintaining it.

Reply:
Done!